File: flix/lib/models/credit.dart
class Credit {
  int? personId;
  String? id;
  String? name;
  String? character;
  String? role;

  Credit({this.personId, this.id, this.name, this.character, this.role});

  Credit.fromJson(Map<String, dynamic> json) {
    personId = json['person_id'];
    id = json['id'];
    name = json['name'];
    character = json['character'];
    role = json['role'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['person_id'] = personId;
    data['id'] = id;
    data['name'] = name;
    data['character'] = character;
    data['role'] = role;
    return data;
  }
}

File: express/src/entities/Credit.ts
import {Entity, Column, PrimaryColumn, ManyToOne} from "typeorm";
import { Title } from "./Title";

@Entity()
export class Credit {
  @PrimaryColumn()
  personId: number;

  @ManyToOne(type => Title, title => title.id) title: Title
  id: number

  @Column()
  name: string;

  @Column()
  character: string;

  @Column()
  role: number;
}